README.md

```diff
@@ -85,14 +85,14 @@ export NODEREPORT_VERBOSE=yes|no
 
 ## Examples
 
-To see examples of NodeReports generated from these events you can run the
-demonstration applications provided in the nodereport github repository. These are
+To see examples of reports generated from these events you can run the
+demonstration applications provided in the node-report github repository. These are
```

Comment:
I have a wider issue, that the same README.md gets used on the npm site and on the github repo, but the information I'd like to have on those two sites is different (the audience is different). So on github I'd like to document the build process, tests and demos, but that's not relevant to the npm user (e.g. they don't have the tests).

Maybe I could put the extra developer info on a page on the github repo wiki, and have a link to that from the README. Or add additional .md files in a docs directory. I wonder what other projects do for this.